pkg/client/clientset.go

```diff
@@ -53,41 +52,50 @@ func (s ClientSet) Entities() *dtclient.DynatraceClient {
 	return s.dtClient
 }
 
-func CreateClientSet(url string, auth manifest.Auth) (*ClientSet, error) {
-	var dtClient *dtclient.DynatraceClient
-	var autClient *automation.Client
-	var err error
+func CreateClassicClientSet(url string, token string) (*ClientSet, error) {
+	concurrentRequestLimit := environment.GetEnvValueIntLog(environment.ConcurrentRequestsEnvKey)
+
+	dtClient, err := dtclient.NewClassicClient(
+		url,
+		token,
+		dtclient.WithAutoServerVersion(),
+		dtclient.WithClientRequestLimiter(concurrency.NewLimiter(concurrentRequestLimit)),
+	)
+	if err != nil {
+		return nil, err
+	}
 
+	return &ClientSet{
+		dtClient:  dtClient,
+		autClient: nil,
+	}, nil
+}
+
+type PlatformAuth struct {
+	ClientID, ClientSecret, Token, TokenURL string
```

Comment:
nit: For clarity of the interface it might make sense to clearly differentiate the Token and TokenURL, currently it looks like they relate to each other. 

Maybe a sub-struct called 

```
OAuth {
  ClientID, ClientSecret, TokenURL
} 
```

and calling the `Token` - `ClassicAPIToken`? 

Or just `ClassicAPIToken` and `OauthTokenURL` ?

Reply:
I've changed it to just:
```go
type PlatformAuth struct {
	OauthClientID, OauthClientSecret, OauthTokenURL string
	Token                                           string
}
```
fine?